### ISSUE TITLE
wp plugin is-installed - not returning anything

### ISSUE THREAD

**Issue by gnowland** [wp-cli/wp-cli#2449]
Tried running `wp plugin is-installed` on multiple real and made up plugin names, nothing returned either way, just a blank prompt.

Other "wp" functions work fine.

Debug:
```
$ wp plugin --debug is-installed hello
Debug: No readable global config found (0.007s)
Debug: No project config found (0.008s)
Debug: ABSPATH defined: /var/www/milnor.com/ (0.022s)
Debug: Begin WordPress load (0.022s)
Debug: wp-config.php path: /var/www/milnor.com/wp-config.php (0.022s)
Debug: Loaded WordPress (0.346s)
Debug: Running command: plugin is-installed (0.346s) 
```

**Comment by danielbachhuber** [comment]
`wp plugin is-installed` returns an exit code, no output:

```
salty-wordpress ➜  wp-api  wp plugin is-installed foo
salty-wordpress ➜  wp-api  echo $?
1
salty-wordpress ➜  wp-api  wp plugin is-installed wp-api
salty-wordpress ➜  wp-api  echo $?
0
```

**Comment by gnowland** [comment]
Ah, alright.

Expected output should probably be added to the man page 😉

On Thu, Feb 4, 2016 at 5:25 PM Daniel Bachhuber <notifications@github.com>
wrote:

> Closed #2449 <https://github.com/wp-cli/wp-cli/issues/2449>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wp-cli/wp-cli/issues/2449#event-539961990>.
>
-- 
--
*Gifford Nowland*
(302) 898-4597 | gnowland@gmail.com
http://www.giffordnowland.com
